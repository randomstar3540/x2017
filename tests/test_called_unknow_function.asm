FUNC LABEL 0
    MOV STK A VAL 8
    MOV REG 0 STK A
    CAL VAL 2
    RET
FUNC LABEL 1
    MOV REG 1 VAL 249
    RET
