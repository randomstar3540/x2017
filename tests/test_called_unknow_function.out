ERROR: Function called do not exist!
